app/workers/data_request_worker.rb

```diff
@@ -0,0 +1,46 @@
+require 'uploader'
+
+class DataRequestWorker
+  include Sidekiq::Worker
+  sidekiq_options retry: 5
+  sidekiq_retry_in do |count|
+    (count ** 8) + 15 + (rand(30) * count + 1)
+  end
+
+  attr_accessor :path
+
+  def perform(request_id)
+    request = DataRequest.find(request_id)
+    return unless request.status == DataRequest::PENDING
+
+    self.path = "tmp/#{request.id}.csv"
```

Comment:
currently i'm putting the files up to S3, so the files are only transient. if we're going to serve the files directly up from caesar then yeah, that makes sense to me, but i wasn't planning to do it that way.